src/DotNest.Core.SDK.Web/DotNest.Core.SDK.Web.csproj

```diff
@@ -0,0 +1,34 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+    </PropertyGroup>
+  
+    <ItemGroup>
+      <PackageReference Include="Lombiq.Analyzers" Version="2.1.1-alpha.0" />
```

Comment:
Let's move this to the other ItemGroup where you reference the other Lombiq libraries.